### PR TITLE
Add mergify rules for backporting

### PR DESCRIPTION
**Pull request by fmauch**
This should enable us to trigger backmerges by simply attaching a label to PRs on main.

### CODE REVIEW

**Review by RobertWilbrandt — CHANGES_REQUESTED** [review]
Do we want an additional ```assignees: ["{{ author }}"]``` rule?

File: .github/mergify.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+pull_request_rules:
+  - name: Backport to galactic branch
+    conditions:
+      - base=main
+      - "label=backport-galactic"
```
[the remark this answers is not in the document]
Comment (fmauch) [reply .github/mergify.yml:5]: That's how I have seen it, but looking at the docs, it might be better to do it as expected...

**Comment by fmauch** [comment]
> Do we want an additional `assignees: ["{{ author }}"]` rule?

I wouldn't necessarily the author of a PR a suitable assignee for a backport. I'd suggest that we assign these from case to case.

**Review by RobertWilbrandt — APPROVED** [review]
LGTM

**Pull request merged**